README.adoc

```diff
@@ -34,9 +34,46 @@ You will develop Arquillian tests that use JUnit as the runner and build your te
 
 You will work with an `inventory` microservice, which stores information about various systems. The `inventory` service communicates with the `system` service on a particular host to retrieve its system properties and store them. You will develop functional and integration tests for the microservices. You will also learn about the Maven and Liberty configurations so that you can run your tests on Open Liberty with the Arquillian Liberty Managed container.
 
+// Getting Started
 [role="command"]
 include::{common-includes}/gitclone.adoc[]
 
+=== Try what you'll build
+
+To try out the application, navigate to the `finish` directory and run the following commands:
+
+[role="command"]
+----
+cd finish
```

Comment:
do they run this command after they navigate to finish? wouldn't this put them in `finish/finish` ?

Reply:
No. So, you suggest to remove `cd finish`?

Reply:
I removed `cd finish` from the commands